### PR TITLE
Draft: Feat/awicm3 v3.2.2

### PR DESCRIPTION
**Pull request by JanStreffing**
This branch shall change the xios default remapping from 2nd to 1st order. @mandresm @alxthor.

So far I just changed the defaults. Better would be to be able to add the relevant option with add_namelist_changes for version 3.2.2 and greater.

This does not work because it's xml. I would like to explore once more if we can't make xml changes happen.

### CODE REVIEW

**Comment by mandresm** [comment]
Hi @JanStreffing,

> I would like to explore once more if we can't make xml changes happen.

Does this mean you're going to be developing that feature or that you'd like us to develop that feature?

Kind regards,
Miguel

**Review by mandresm — APPROVED** [review]

File: configs/components/fesom/fesom-2.5.yaml [hunk 1]
```diff
@@ -35,9 +35,12 @@ choose_version:
     icb_code: true
     namelist_dir: "${model_dir}/config/"
 
-  '2.5-awicm-3.2':
+  "2.5-awicm-3.2":
     branch: "AWI-CM3_v3.2"
 
+  "2.5-2":
```
Comment (mandresm) [inline configs/components/fesom/fesom-2.5.yaml:41]: Shouldn't this be `2.5.2`?

Comment (mandresm) [reply configs/components/fesom/fesom-2.5.yaml:41]: Forget about it, I have seen the issue #1184

**Review by mandresm — APPROVED** [review]

**Comment by mandresm** [comment]
#bump

**Comment by mandresm** [comment]
#approve-changes

**Comment by JanStreffing** [comment]
Oh, this was merged already, ups. I wanted to look into the xml thing first. As is, this will affect all AWI-CM3 simulations, regardless of version number. Time for a hotfix where I duplicate the xml folders. This was supposed to be a draft still :D

**Pull request merged**